liblocation/src/main/java/com/mapbox/android/core/location/LocationPendingIntent.java

```diff
@@ -0,0 +1,8 @@
+package com.mapbox.android.core.location;
+
+import android.app.PendingIntent;
+
+interface LocationPendingIntent {
+
+  PendingIntent retrievePendingIntent();
```

Comment:
I'm not seeing the `IntentService` setup anymore, do we still need this interface?

Reply:
We are still using it for the BroadcastReceiver